tests/shell/check_resolver.sh

```diff
@@ -190,11 +190,11 @@ check_resolver user b x/a @KDB_DB_HOME@/@KDB_DB_USER@/x/a
 check_resolver user b /a @KDB_DB_HOME@/a
 
 OD=`pwd`
-cd /tmp # hopefully no @KDB_DB_DIR@ is in /tmp
-check_resolver dir b /a /tmp/a
-check_resolver dir b /a/b /tmp/a/b
-check_resolver dir b a /tmp/@KDB_DB_DIR@/a
-check_resolver dir b a/b /tmp/@KDB_DB_DIR@/a/b
+cd $TMPPATH # hopefully no @KDB_DB_DIR@ is in $TMPPATH
```

Comment:
Why is `@KDB_DB_DIR@` in `$TMPPATH` a problem?

What about using `mktemp -d` instead? Maybe we do not need the OSTYPE check at all, then? It would also solve the problem of `@KDB_DB_DIR@` in `$TMPPATH`. (but obviously we would need a cleanup-routine then)